webapp/channels/src/components/drafts/scheduled_post_list/scheduled_post_list.tsx

```diff
@@ -45,6 +46,11 @@ export default function ScheduledPostList({
     const targetId = query.get('target_id');
     const targetScheduledPostId = useRef<string>();
 
+    const dispatch = useDispatch();
+    useEffect(() => {
+        dispatch(fetchMissingChannels(scheduledPosts.map((post) => post.channel_id)));
```

Comment:
Not that it needs to change, but if you wanted to move this back to the `DraftRow`, you could do that now. The `DataLoader` would sort out requests from the multiple components for you so that it should behave virtually the same way it does with a single `fetchMissingChannels` call here